tests/test_api.py

```diff
@@ -43,13 +44,15 @@
 # Yuck, globals
 connection = table = None
 
+if sys.version_info >= (3,):
+    xrange = range
```

Comment:
I feel the same way, though I'm generally cautious about overwriting a module's reference to a built-in in another developer's code.